features/step_definitions/web_steps.rb

```diff
@@ -62,6 +62,14 @@ def with_scope(locator)
   end
 end
 
+Then "they see {}" do |selector|
+  assert_selector(:css, selector_for(selector), wait: 0)
+end
+
+Then "they don't see {}" do |selector|
+  assert_not has_css?(selector_for(selector), wait: 0)
+end
```

Comment:
I'm pretty sure we already have steps to check page for string values. Maybe make sure step names contain a clue that this is a CSS selector?

Reply:
Yes, these will be cleaned in time. There is A LOT of repetition in general